src/pages/experiments/[experimentId]/plots-and-tables/marker-heatmap/index.jsx

```diff
@@ -59,9 +59,22 @@ const MarkerHeatmap = ({ experimentId }) => {
 
   useEffect(() => {
     if (louvainClustersResolution && config) {
-      dispatch(loadMarkerGenes(experimentId, louvainClustersResolution, plotUuid, config.numGenes));
+      dispatch(loadMarkerGenes(
```

Comment:
you could add a `louvainClustersResolutionRef` as is done in data exploration so that if `configureEmbedding` in the redux store is somehow overwritten but the clustering doesn't change then we don't re run `loadMarkerGenes`.

(I'm tlaking about this part:
  ```
useEffect(() => {
    if (louvainClustersResolution
      && louvainClustersResolutionRef.current !== louvainClustersResolution
    ) {
      louvainClustersResolutionRef.current = louvainClustersResolution;
      dispatch(loadMarkerGenes(experimentId, louvainClustersResolution, COMPONENT_TYPE));
    }
  }, [louvainClustersResolution]);
```